src/pages/dashboardOverview/components/notActiveProductsSection/NotActiveProductsSection.tsx

```diff
@@ -37,6 +48,7 @@ export default function NotActiveProductsSection({ party, products }: Props) {
                     (subProduct) => !party.products.some((us) => us.productId === subProduct.id)
                   )))
           )
+          .filter((p) => p.id !== 'prod-pn' || isOnboardingAllowedInProdPN(party?.category))
```

Comment:
You are filtering the SEND product always and in any case, you should filter it only if the categories are not L6,L4,L45, if the organization belongs to one of these categories and has not subscribed to the SEND product, it will have to see the card in the available products section